src/pages/Dashboard/Apps/Info/index.jsx

```diff
@@ -11,6 +11,7 @@ import {
 import Layout, { Card } from 'components/Layout';
 import DetailTabs from 'components/DetailTabs';
 
+import PropTypes from 'prop-types';
```

Comment:
third party lib should lift up, place it after react

Reply:
ok